src/org/labkey/cds/CDSController.java

```diff
@@ -226,16 +226,33 @@ public void setUserProperties(JSONObject userProperties)
 
     private static final String ANALYTICS_USER_GROUP = "Active CAVD Member";
 
+    @RequiresNoPermission
```

Comment:
should require read permission for updates

Reply:
Done!